terraform/spokes/main.tf

```diff
@@ -380,10 +388,37 @@ module "eks" {
           # Reference docs https://docs.aws.amazon.com/eks/latest/userguide/cni-increase-ip-addresses.html
           ENABLE_PREFIX_DELEGATION = "true"
           WARM_PREFIX_TARGET       = "1"
-        }
+        },
+        enableNetworkPolicy = "true"
       })
     }
+    aws-ebs-csi-driver = {
```

Comment:
We should add Cloudwatch container insights also

Reply:
Yes this part will be updated on second revision after the core changes are safe